frontend/src/components/profiles/UserProfile.js

```diff
@@ -1,14 +1,76 @@
 import React from "react";
-import User from "./User/User.js";
+// import User from "./User/User.js";
+import "../../App.css";
```

Comment:
App.css styles globally no need to import in any file other than App.js

Reply:
okay I will update it